modules/learn/pages/clusters-and-availability/rebalance.adoc

```diff
@@ -90,14 +90,14 @@ A _move_ counts toward this restriction only when in the _backfill_ phase, as de
 The move may be of either an _active_ or a _replica_ vBucket.
 A node's participation in the move may be as either a source or a target.
 
-For example, if a node is at a given time the source for two moves in backfill phase, and the target for two additional moves in backfill phase, and the setting stands at `4`, the node may participate in the backfill phase of no additional moves, until at least one of its current moves has completed its backfill phase.
+For example, if a node is at a given time the source for two moves in backfill phase, and is the target for two additional moves in backfill phase, and the setting stands at `4`, the node may participate in the backfill phase of no additional moves, until at least one of its current moves has completed its backfill phase.
 
 The setting may be established by means of Couchbase Web Console, the Couchbase CLI, or the REST API: for information, see the page on xref:manage:manage-settings/general-settings.adoc[General] Settings.
 
 A higher setting may _increase_ rebalance performance, at the cost of higher resource consumption; in terms of CPU, memory, disk, and bandwidth.
 Conversely, a lower setting may _decrease_ rebalance performance, while freeing up such resources.
 Note, however, that rebalance performance can be affected by many additional factors; and that in consequence, changing this parameter may not always have the expected effects.
-Note also that a higher setting, due to its additional consumption of resources, may _decrease_ the performance of other systems hosted by the cluster.
+Note also that a higher setting, due to its additional consumption of resources, may _decrease_ the performance of other systems, including the Data Service.
```

Comment:
IMO, degrade is a better word here that decrease.

Reply:
Okay, I've changed accordingly. Thanks!